elasticsearch/prep-install.origin

```diff
@@ -0,0 +1,15 @@
+#reconcile upstream to downstream
+es_source=/usr/share/elasticsearch
+for file in $(ls $es_source)
```

Comment:
isn't it a bit dangerous if anything in `$es_source` has a space in the name for instance? Why not use bash globbing?